src/compute/src/render/context.rs

```diff
@@ -760,29 +1034,26 @@ where
             if !self.arranged.contains_key(&key) {
                 // TODO: Consider allowing more expressive names.
                 let name = format!("ArrangeBy[{:?}]", key);
```

Comment:
To make debugging easier, it would be good to include the specialized nature of the arrangement in the name.

Reply:
This is already done in `specialized_arrange` as a type tag appended to the name in case specialization actually takes place. I did not want to do it here because the errors arrangement is never specialized. So I would find it confusing to add type-specialization information to its name.